.changeset/new-doors-fold.md

```diff
@@ -0,0 +1,5 @@
+---
+"@spectrum-css/combobox": minor
+---
+
+Combobox now includes a `read-only` state. Add the `is-readOnly` class to the Combobox to enable this feature. By doing so, this also activates the `readonly` attribute in the `Textfield` component. To ensure the intended user experience please make sure the readonly state in the textfield is enabled by adding the `is-readOnly` class and `readonly` attribute on the input element for this component as well.
```

Comment:
```suggestion
Combobox now includes a `read-only` state. Add the `is-readOnly` class to the Combobox to enable this feature. To ensure the intended user experience please make sure the readonly state in the textfield is enabled by adding the `is-readOnly` class and `readonly` attribute on the input element for this component as well.
```
The documentation is in context of implementations using our shipped CSS, not our Storybook.

Reply:
I also just inspected the component markup and saw that `is-readOnly` is added to `spectrum-Textfield`, and the `readonly` attribute is added to the input within it. So both this and the docs need a little adjustment to clarify; right now it sounds like both are added to the input.